src/logics/uniswap-v3/configs.ts

```diff
@@ -42,6 +42,15 @@ export const configs: univ3.Config[] = [
     feeAmounts: [FeeAmount.LOWEST, FeeAmount.LOW, FeeAmount.MEDIUM, FeeAmount.HIGH],
     bases: [polygonTokens.WETH, polygonTokens.USDC, polygonTokens.DAI, polygonTokens.USDT, polygonTokens.WBTC],
   },
+  {
+    // https://github.com/Uniswap/interface/blob/v4.270.6/src/constants/routing.ts#L97
+    chainId: common.ChainId.base,
+    factoryAddress: '0x33128a8fC17869897dcE68Ed026d694621f6FDfD',
+    quoter: { address: '0x3d4e44Eb1374240CE5F1B871ab261CD16335B76a', isV2: true },
+    swapRouterAddress: '0x2626664c2603336E57B271c5C0b26F421741e481',
```

Comment:
Isn't the one on the base also SwapRouter02?

Reply:
woops. The name on the [deployment page](https://docs.base.org/contracts/) is swapRouter but the contract name is swapRouter02. Will remove this one.

Reply:
I think this page is more accurate https://docs.uniswap.org/contracts/v3/reference/deployments